h/traversal/__init__.py

```diff
@@ -63,6 +63,7 @@
 from h.traversal.annotation import AnnotationContext, AnnotationRoot
 from h.traversal.group import GroupContext, GroupRequiredRoot, GroupRoot
 from h.traversal.group_membership import (
+    AddGroupMembershipContext,
```

Comment:
We tend to import names from `h/*/*/*.py` files into `h/*/__init__.py` to make imports easier. It means you can do this:

```python
from h.traversal import Foo, Bar, Gar
```

Instead of having to do this:

```python
from h.traversal.foo import Foo
from h.traversal.bar import Bar
from h.traversal.gar import Gar
```

It also means the importer doesn't know about the internal structure of `h/traversal/` so we can reorganise the internal files without breaking any importers.